Source/Extensions/Blazorise.Video/Video.razor.cs

```diff
@@ -728,12 +730,12 @@ public Task NotifyQualityChange( int? quality )
     /// <summary>
     /// Sent when a seek operation begins.
     /// </summary>
-    [Parameter] public Func<Task> Seeking { get; set; }
+    [Parameter] public Func<double, Task> Seeking { get; set; }
```

Comment:
Since we are touching the apis, why don't these just use regular event callbacks? Any special reason?

Reply:
Regular event callbacks refreshes the page, and func is not. I want to avoid refreshing whenever necessary. Especially with heavy components like video.

Reply:
Gotcha, nice thinking. Although I'm not entirely sure about the veracity of "Regular event callbacks refreshes the page".
As far as I know, It calls `StateHasChanged` which not necessarily means that the page will refresh. The component specifically will be checked for changes and then refreshed, is this not the case?

Reply:
Yes. It will call StateHasChanged but it will still refresh everything on the same level of rendering tree, and bellow.

Reply:
Gotcha, my point. So I would say that it would really be doing almost nothing at all since Video won't inherit children?  Anyway this approach is indeed better nevertheless, just wanted to be sure that we were on the same page about the statement :
> "Regular event callbacks refreshes the page"